aip/general/0128.md

```diff
@@ -75,14 +75,13 @@ is a comprehensive reference to declarative-friendly guidance in other AIPs:
 - Resources **should** provide change validation: see AIP-163.
 - Resources **should not** implement soft-delete. If the id cannot be re-used,
   the resource **must** implement soft-delete and the undelete RPC: see AIP-164
-
-
-### Annotations
-
-See [AIP-148 annotations][].
+- Resources **must** include a
+`map<string, string> annotations` field to allow clients to store small amounts
+of arbitrary data: see AIP-148
```

Comment:
Would you say that this field has a similar intent as https://stripe.com/docs/api/metadata

Reply:
They are intended to have the same purpose as https://kubernetes.io/docs/concepts/overview/working-with-objects/annotations/

Reply:
I can only agree with both comments above :)

Reply:
I understand the usage of `annotations` in the context of Kubernetes, but I am trying to find the usefulness from the AIP perspective outside that given Domain.

It should be a less technical-focused product and more general in the public domain (fintech, education, etc.).

Kind of related to https://github.com/aip-dev/google.aip.dev/pull/1288#discussion_r1433368922

I think the topic of transient data and/or user-controlled (external clients own the data) instead of system-controlled (internal to systems, not intended outside programmers that own the system).